tests/test_selection/test_drop_features.py

```diff
@@ -26,6 +26,13 @@ def test_drop_1_variable(df_vartypes):
     assert type(X) == pd.DataFrame
     pd.testing.assert_frame_equal(X, df)
 
+    # checking features_to_drop has been correctly cast to list of str internally
+    transformer = DropFeatures(features_to_drop='Marks')
```

Comment:
Hi, this looks good, but it took me a while to understand what this test was about, and why it is different from that in line 8.

Could we, instead of making it part of the current test, have a different test, just for this issue, with a clear name of what is being tested, and also in a comment add a link to the issue or PR?